### PR TITLE
Omega soap

### PR DESCRIPTION
**Pull request by itseasytosee**
## About The Pull Request
Legend says that the Omega soap can be created by only the most skilled chemist cursed with forsaken knowledge.
The riddle of its creation goes...

_Essence of **potato**. Juice, not grind.
A **lizards**' tail, turned into **wine**.
**powder of monkey**, to help the workload.
Some **Krokodil**, because meth would explode.
**Nitric acid** and **Baldium**, for organic dissolving.
A cup filled with **Hooch**, for sinfull absolving
Some **Bluespace Dust**, for removal of stains.
A syringe full of **Pump-up,** it security's bane.
Add a can of **Space Cola,** because we've been paid.
**Heat** as hot as you can, let the soap be your blade._

**Ten units of each regent create a soap that could topple all others.**

image

## Why It's Good For The Game
It takes a very long amount of time to create and requires corporation with botany the bartender and various chemicals from vending machines. If a janitor can pull it off he deserves the sub space-cleaner cleaning device.

## Changelog
:cl: itseasytosee
add: The Omega soap chemical recipe
/:cl:

### CODE REVIEW

**Comment by 81Denton** [comment]
https://www.youtube.com/watch?v=IuysY1BekOE

Me while reading the recipe at 6AM

**Comment by improvedname** [comment]
>you don't need lye
B-but candles lye. S-so does gibsoap why not this aaaa

File: code/game/objects/items/clown_items.dm [hunk 1]
```diff
@@ -73,6 +73,13 @@
 	icon_state = "soapsyndie"
 	cleanspeed = 5 //faster than mop so it is useful for traitors who want to clean crime scenes
 
+/obj/item/soap/omega
+	name = "omega soap"
+	desc = "The most advanced soap known to mankind."
+	icon_state = "soapomega"
+	cleanspeed = 3 //Only the truest of mind soul and body get one of these
+	uses = 301
```
Comment (XDTM) [inline code/game/objects/items/clown_items.dm]: I'd put the uses at double the regular amount at least, maybe even infinite.

**Comment by XDTM** [comment]
Potential ideas to go with the bluespace theme: make the soap slip into bluespace like botany bananas, or maybe allow ranged cleaning.

**Comment by itseasytosee** [comment]
This wasn't intended to have a bluespace theme. Its blue because about a third or the regents are blue to blue-ish purple.

**Comment by itseasytosee** [comment]
>you don't need lye
B-but candles lye. S-so does gibsoap why not this aaaa

Lye is for cleaning. This soap purifies.

**Comment by improvedname** [comment]
> > you don't need lye
> > B-but candles lye. S-so does gibsoap why not this aaaa
> 
> Lye is for cleaning. This soap purifies.

No lye is so it can become solid

**Comment by itseasytosee** [comment]
Monkey cubes become solid. So dose space cola.

File: code/game/objects/items/clown_items.dm (hunk 1 above)
Comment (ExcessiveUseOfCobblestone) [reply code/game/objects/items/clown_items.dm]: it already has extremely fast cleanspeed

**Comment by Shadowflame909** [comment]
I prefer orange soda

File: code/game/objects/items/clown_items.dm (hunk 1 above)
Comment (ATH1909) [reply code/game/objects/items/clown_items.dm]: Yeah, give it an infinite number of uses.

Comment (XDTM) [reply code/game/objects/items/clown_items.dm]: for powerlevel consider that this still won't beat a floor buffer or a janiborg, unless all the mess is specifically under tables or on walls

**Comment by itseasytosee** [comment]
It was sponsored. A poem like that doesn't come cheap.

File: code/game/objects/items/clown_items.dm (hunk 1 above)
Comment (itseasytosee) [reply code/game/objects/items/clown_items.dm]: I do not plan on giving it an absurd number or infinite amount of uses.

Comment (itseasytosee) [reply code/game/objects/items/clown_items.dm]: You will not need to use this more than 301 times and if you do just whip up a double batch in a bluespace beaker.

**Comment by CRITAWAKETS** [comment]
Needs a higher temperature, forcing the chemist to get semi-creative with how to heat his thing

ie; use a lighter

**Comment by RaveRadbury** [comment]
This is like a philosophers' stone for lizard janitors. Great power, but at what price?

**Comment by cacogen** [comment]
this is going to make fuck all difference to the never-ending tedium that is playing janitor but whatever

**Comment by RaveRadbury** [comment]
> this is going to make fuck all difference to the never-ending tedium that is playing janitor but whatever

_This guy plays janitor for the fast-paced no-talk cleaning action!_

**Comment by Bawhoppen** [comment]
Tie-in video: https://www.youtube.com/watch?v=e_ncSippOGY

**Comment by cacogen** [comment]
Hello, /tg/station. Please forgive my unusual method of reaching you this
balmy summer's night. I am on a quiet stroll. It's windy as fuck.

I play janitor to keep the station clean. It's cathartic. Unfortunately for
a number of reasons it's a losing battle. It doesn't have to be, it just
always has been.

From what I can tell this soap is shit for the effort involved in making
it. I'm not sure what I'd prefer if it were up to me, but someone's idea of
it being bluespace was a start. I say this with no expectation of being
listened to whatsoever.

File: code/game/objects/items/clown_items.dm (hunk 1 above)
Comment (Tlaltecuhtli) [reply code/game/objects/items/clown_items.dm]: give it aoe telephathic cleaning or its gonna be made once or twice then it ll turn into a lazy admins epic event loot

**Comment by cacogen** [comment]
tlal just says it immediately perfectly

**Comment by knacker48** [comment]
Maybe I'm missing something but is the only thing special about this soap is that it cleans fast?

File: code/game/objects/items/clown_items.dm (hunk 1 above)
Comment (ExcessiveUseOfCobblestone) [reply code/game/objects/items/clown_items.dm]: fyi cleaning skill makes it so there's a chance it won't use one of the uses.

**Comment by ExcessiveUseOfCobblestone** [comment]
it's really hard to beat things that can clean an entire tile simply by walking over it with no delay.

**Comment by itseasytosee** [comment]
I wasn't really thinking about balance at the time. I'm not gonna buff it uncreatively.
The only upside this thing has on jani-cart/janiborg is for tables, clothing, weapons, messy walls, etc.
But if you are using it for that purpose it is really effective compared to soap/mop.

**Comment by cacogen** [comment]
>it's really hard to beat things that can clean an entire tile simply by
walking over it with no delay.

Those aren't the problem!

**Comment by Fikou** [comment]
make it clean anything that its on, so if you throw it across the hallway all of the tiles it was on become clean

**Comment by ExcessiveUseOfCobblestone** [comment]
I disagree but I really have no interest in discussing the balance around janitors/messes

**Review by kriskog — COMMENTED** [review]
Pretty cool, looks allright but I would like for there to be some fluff to hint at this in game for the curios janitor. You already have a nice poem, get it on a piece of fluff paper and hide it somewhere.

You also have a merge conflict.

**Comment by itseasytosee** [comment]
I have an issue regarding my original repo being deleted. I will redo the code tomorrow now that all the thinking is out of the way

**Pull request closed without merging**